src/test/kotlin/com/hibob/project/service/FeedbackServiceTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.hibob.project.service
+
+import com.hibob.project.dao.*
+import jakarta.ws.rs.NotFoundException
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import java.sql.Timestamp
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class FeedbackServiceTest {
+    private val feedbackDao = mock<FeedbackDao>()
+    private val feedbackService = FeedbackService(feedbackDao)
+    val companyIdTest = 1234L
+
+    @Test
+    fun `get all feedbacks where feedback list is empty and throws exception`() {
+        val expectedMessage = assertThrows<NotFoundException> { feedbackService.getAllFeedbacks(companyIdTest) }
+        assertEquals("There are no feedbacks for this company", expectedMessage.message)
```

Comment:
you don't really need to validate the excpetion message, this make the code highly coupled with the tests

Reply:
We have learned to validate the exception message this way